index.html

```diff
@@ -2250,6 +2329,36 @@ <h1>Core Representations</h1>
 for more information.
     </p>
 
+    <section>
+      <h2>Representation Requirements</h2>
+      <p>
+The <a href="#data-model">data model</a> provided in this specification
+supports being serialized into a variety of existing representations. Some
+applications might require the creation of a new representation. All
+representations require the following:
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding for all
+property names and their associated values as defined in this specification.
+This enables anything that can be represented in the <a>DID document</a> data
+model to also be represented in a compliant representation.
```

Comment:
```suggestion
A representation MUST define an unambiguous encoding and decoding for all
data model <a href="#data-model">data types</a> defined in this specification.
This enables anything that can be represented in the <a>DID document</a> data
model to also be represented in a compliant representation.
```

Reply:
Perhaps this is what @jricher is looking for?

Reply:
More like this, translating the property names is also important (this is what lets JSON-LD have additional contexts and namespaces, and CBOR have tags for property names):

```suggestion
A representation MUST define an unambiguous encoding and decoding for all
property names and all data model <a href="#data-model">data types</a> 
as defined in this specification.
This enables anything that can be represented in the <a>DID document</a> data
model to also be represented in a compliant representation.
```